src/seedu/addressbook/AddressBook.java

```diff
@@ -439,15 +439,18 @@ private static String getMessageForSuccessfulAddPerson(HashMap<String, String> a
     }
 
     /**
-     * Finds and lists all persons in address book whose name contains any of the argument keywords.
-     * Keyword matching is case sensitive.
+     * Finds and lists all persons in address book whose name starts with contains any of the argument keywords.
+     * Keyword matching is not case sensitive.
```

Comment:
generally we try not to use `is not` if the antonym exist, e.g. case insensitive
because it confuse other programmers when used in boolean 

e.g. `boolean isNotChanged = false;` << double negative makes it confusing